### PR TITLE
0.2.0-beta1

### PR DESCRIPTION
**Pull request by slvrtrn**
## Summary
See #165 #189  

## Checklist
- [x] A human-readable description of the changes was provided to include in CHANGELOG

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,12 +1,43 @@
...
+and [WebStream](https://developer.mozilla.org/en-US/docs/Web/API/Streams_API) APIs)
+with no Node.js modules in the common interfaces. No polyfills are required.
+
+It is now possible to implement new custom connections on top of `@clickhouse/client-common`.
+
+The client was refactored into three packages:
+
+- `@clickhouse/client-common`: all possible platform-independent code, types and interfaces
+- `@clickhouse/client-browser`: new "browser" (or non-Node.js env) connection, uses native fetch.
+  Depends on the `@clickhouse/client-common` package.
```
Comment (mshustov) [inline CHANGELOG.md]: IMO `Depends on the `@clickhouse/client-common` package.` can be removed, it's clear from the context

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,12 +1,43 @@
...
+- `@clickhouse/client`: Node.js connection as it was before. Depends on the `@clickhouse/client-common` package.
+
+### Node.js client breaking changes
+
+- Log level configuration parameter is now explicit instead of `CLICKHOUSE_LOG_LEVEL` environment variable.
+- `query` return type signature changed to is `BaseResultSet<Stream.Readable>` (no functional changes)
+- `exec` return type signature changed to `ExecResult<Stream.Readable>` (no functional changes)
+- `insert<T>` params argument type changed to `InsertParams<Stream, T>` (no functional changes)
+- Experimental `schema` module is removed
+-
```
Comment (mshustov) [inline CHANGELOG.md]:
```suggestion
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,12 +1,43 @@
...
+The client was refactored into three packages:
+
+- `@clickhouse/client-common`: all possible platform-independent code, types and interfaces
+- `@clickhouse/client-browser`: new "browser" (or non-Node.js env) connection, uses native fetch.
+  Depends on the `@clickhouse/client-common` package.
+- `@clickhouse/client`: Node.js connection as it was before. Depends on the `@clickhouse/client-common` package.
+
+### Node.js client breaking changes
+
+- Log level configuration parameter is now explicit instead of `CLICKHOUSE_LOG_LEVEL` environment variable.
```
Comment (mshustov) [inline CHANGELOG.md:17]: Let's add a comment about a default value

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,12 +1,43 @@
...
+- `insert<T>` params argument type changed to `InsertParams<Stream, T>` (no functional changes)
+- Experimental `schema` module is removed
+-
+
+### Browser client known limitations
+
+- Streaming for select queries works, but it is disabled for inserts (on the type level as well).
+- KeepAlive is disabled and not configurable yet.
+- Request compression is disabled and ignored.
+- No logging support yet.
```
Comment (mshustov) [inline CHANGELOG.md:29]: Let's do in a follow-up, we always can log in console

Comment (slvrtrn) [reply CHANGELOG.md:29]: We have it https://github.com/ClickHouse/clickhouse-js/issues/187

**Pull request merged**